src/components/inline-popup/inline-popup.scss

```
@@ -30,10 +38,13 @@
 
 	border: var(--inline-popup_border-width) solid
 		var(--inline-popup_border-color);
+	// TODO(undefined-css-vars): no similar token found for --icon-size_dense or --icon-size-dense-padding
+	/* stylelint-disable csstools/value-no-unknown-custom-properties */
 	border-radius: calc(
 		var(--inline-popup_border-width) + var(--inline-popup_padding-vertical) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)